src/cmd_line.rs

```diff
@@ -69,6 +71,11 @@ pub fn handle_command_line_arguments() {
                 .long("multiGrid")
                 .help("Enable Multigrid"),
         )
+        .arg(
+            Arg::with_name("frameless")
+            .long("frameless")
+            .help("Enables Frameless Neovide")
```

Comment:
I think maybe a better help message would be "Removes the window frame. Note, might make the window not resizable"

Reply:
ok then